gallery/src/styles/NxTile/NxTileFormExample.tsx

```diff
@@ -28,7 +28,7 @@ export default function NxTileFormExample() {
           <div className="nx-form-group">
             <label className="nx-label">
               <span className="nx-label__text">Label</span>
-              <NxStatefulTextInput validator={validator}/>
+              <NxStatefulTextInput aria-required={true} validator={validator}/>
```

Comment:
As above, if we're going to change the layout I would prefer that we are consistent about whether the `input` is inside or outside the `<label>`